__main__.py

```diff
@@ -91,7 +92,7 @@ def main():
 
             for script in script_list:
                 if script.name:
-                    if args.l!=None:
+                    if args.l is not None:
```

Comment:
@henrykironde The value of l is True when retriever ls is called with the -l flag.